rules/subrepo_rules.build_defs

```diff
@@ -190,6 +201,9 @@ def github_repo(name:str, repo:str, revision:str, build_file:str=None, hashes:st
         build_file = build_file,
         hashes = hashes,
         config = config,
+        headers = {
+            'Authorization': f'token {access_token}'
+        },
```

Comment:
I think we'd need to guard against this otherwise we'll be passing an incomplete authorisation header to public repos

Reply:
done